### DISCUSSION TITLE
does this software let you connect a bluetooth speaker and Home assistant?

### DISCUSSION THREAD

**Discussion by comet424** [sle118/squeezelite-esp32#418] [General]
so i was googling and came across this software and skimming i not sure if it works what i want..

what i basiclly looking for is since Home assistant Bluetooth built in feature is broken... and i wanna connect a bluetooth speaker to HA  to send text to speech automation

id like to know if i can just simple use a ESP32-wroom32  connect to HA and use it as a Bluetooth connection to Speaker and paired and stream the audio  dont wanna solder anything  jsut want it to be a blue tooth server... does this software do that?  or do you need to solder stuff etc

**Comment by Renber78** [comment]
This project is a client for Squeezebox, it does not correspond at all to what you want to do.

Communication is done here via Wifi, the BT functionality is simply a little extra added to the project.

Afterwards, yes it is possible to make a notification device with this project and HA but the line to follow is not the one you follow. A Squeezebox server and some programming knowledge will be required.

Attached is another ESP32 project under Arduino: *ttps://github.com/iovidius/BarryBox
You will be able (still via Wifi) to send commands from HA (MQTT) in order to trigger audible alarms or TTS
This seems like a good start to what you want to do.
It will be less heavy, cleaner and more reliable than BT.

**Comment by jsarcand** [comment]
lms or music assistant can be integrated with home assistant.  Then, the endpoints can be used to send text to speech.  
You will need to install a LMS server, or Music Assistant 2.0.

Sync esp32 to bluetooth, but then your speaker will need to stay open, as it might not turn back on by itself when requested.  YMMV but in theory it should work.

**Comment by formatBCE** [comment]
I have this, with Music Assistant. Works very well.
